src/main/java/org/prebid/server/json/deserializer/BidTypeOrdinalDeserializer.java

```diff
@@ -0,0 +1,50 @@
+package org.prebid.server.json.deserializer;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+
+import java.io.IOException;
+
+public class BidTypeOrdinalDeserializer extends StdDeserializer<BidType> {
```

Comment:
Why we need this one, as i remember if someone will make this field with impossible value than we will have clear massage for bad input

Reply:
Removed.